notebooks/MAST/TESS/interm_tesscut_astroquery/interm_tesscut_astroquery.ipynb

```diff
@@ -0,0 +1,507 @@
+    "<p>\n",
+    "Note, the request is returned in a json format. The 'results' key contains an array of dictionaries with the information we are looking for.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "coord = SkyCoord(Ra, Dec, unit = \"deg\")\n",
+    "\n",
+    "sector_table = Tesscut.get_sectors(coord)\n",
+    "print(sector_table)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The resulting table shows that this target was observed in two different sectors using the same camera and CCD."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"tesscut_ID\"></a>\n",
+    "## Request pixel timeseries cutout from TESS FFIs\n",
+    "Astrocut is the tool that runs the cutout service around the RA and Dec that were requested. It delivers a zipped file containing a cutout for each set of FFIs as listed above. It is also possible to request only one sector using the \"sector\" parameter.  \n",
+    "<br>\n",
+    "For tesscut, ***x*** refers to the CCD columns and ***y*** refers to the CCD rows. Distance can be input in a variety of units, I chose pixels (\"px\")."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hdulist = Tesscut.get_cutouts(coord, 20)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Notice that this returns a **list** of HDUlist objects, one for each of our files.  HDUlist objects are the same thing you get back as if you downloaded the file and then run the following on the file: \n",
+    "hdu =  astropy.io.fits.open(FITS_file_name)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hdulist[0].info()\n",
+    "hdulist[0][0].header['SECTOR']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hdulist[1].info()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[Top of Page](#title_ID)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"plot_image\"></a>\n",
+    "## Plot the First Image of the Time Series\n",
+    "Here we show you some of the information found in the cutout. The format is almost identical to a target pixel file. You might read through the target pixel file tutorial if you are not familiar with that file type.  \n",
```

Comment:
"The format is almost identical to a target pixel file." I am confused, I thought the format *was* a TPF? Do you almost identical to a TESS pipeline TPF?